apps/bfd-db-migrator/src/main/java/gov/cms/bfd/migrator/app/MigratorApp.java

```diff
@@ -0,0 +1,33 @@
+package gov.cms.bfd.migrator.app;
+
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The main application/entry point for the Database Migration system. See {@link
+ * MigratorApp#main(String[])}.
+ */
+public final class MigratorApp {
+  private static final Logger LOGGER = LoggerFactory.getLogger(MigratorApp.class);
+
+  /**
+   * This method is called when the application is launched from the command line. Currently, this
+   * is a notional placeholder application that produces NDJSON-formatted log output.
+   *
+   * @param args (should be empty. Application <strong>will</strong> accept configuration via
+   *     environment variables)
+   */
+  public static void main(String[] args) {
+    Runnable helloWorld =
+        new Runnable() {
+          public void run() {
+            LOGGER.info("Hello, World!");
+          }
+        };
+    ScheduledExecutorService executor = Executors.newScheduledThreadPool(1);
+    executor.scheduleAtFixedRate(helloWorld, 0, 3, TimeUnit.SECONDS);
+  }
```

Comment:
regardless of the above I think we want to explicitly return 0 since in the future this will return a value based on what happened with the migrations and validations.